### DISCUSSION TITLE
Using bigtext with custom colors

### DISCUSSION THREAD

**Discussion by Celtech** [pterm/pterm#320] [Q&A] [answered]
Hey guys,

Is it possible to use colored big text with a non `Color` color? I.e lets say I want to use a RGB color, how would I do that?

Use case:
I want orange text, orange is not part of the default `fg*` colors.

**Comment by MarvinJWendt** [comment, marked as answer]
Hey @Celtech, @floaust and I will work on this now! I expect the feature to be ready today :)
Thanks for the heads-up, this is not possible now indeed!

**Reply by MarvinJWendt** [reply]
`v0.12.38` is out and contains `pterm.NewLettersFromStringWithRGB(...)`!
FYI: We're planning a breaking change that will move this functions to the `putils` package, so you might look out for it in the next months. Then it'll be `putils.NewLettersFromStringWithRGB(...)`.

**Reply by Celtech** [reply]
Wow! Thank you so much for the lightning fast turn around time! Will test this out later this evening, and report back with the results! 

Also thank you for the amazing package! Loving this since I found it the other night!

**Comment by Celtech** [comment]
Went ahead and tested the new version, it works perfectly as far as I can tell. That said if you use it with a terminal that doesn't support true color such as `terminal.app`, you can get some pretty wild results! haha

Thanks again you guys for your awesome work on this package!

**Reply by MarvinJWendt** [reply]
You're welcome! We'll take a look at the TrueColor Support, normally PTerm should automatically fall back to a close supported color 😅

**Reply by Celtech** [reply]
Is it possible to override the fallback behavior?

e.g:
- lets say I make some big text in the color orange and blue using RGB.
<img width="733" alt="image" src="https://user-images.githubusercontent.com/7949140/157724691-e89b1ebb-9d4c-4ea6-ba99-deeb507ed8b4.png">
- Now let's say that the user's terminal doesn't support true color so it selects a fallback automatically to this blue and yellow color
<img width="1270" alt="image" src="https://user-images.githubusercontent.com/7949140/157724761-976be899-6259-42d1-919f-37187b67a2ed.png">

Would it be possible to make it fall back to just flat white or control what color it falls back to?

**Reply by MarvinJWendt** [reply]
Hi @Celtech, this will be handled in #330!

**Reply by MarvinJWendt** [reply]
This is out now! Let me know if it works! If the `RGB` color is not supported, it will fallback to the set style. Hope this fixes your Issue!
